### PR TITLE
type change rounded icon set

### LINKED ISSUE #159: Project horizontal view icons
The icons in the project horizontal view need to be rounded to fit the look of the icons in all other areas of the project.

Current:
image

They should have the same rounded corners as the other icons:
image

### PR DESCRIPTION
**Pull request by KcPele**
## Description
Icons rounded fixed #159, #158 and fixed some type error
_Concise description of proposed changes, We recommend using screenshots and videos for better description_
Screenshot from 2023-10-30 19-43-09
Screenshot from 2023-10-30 19-42-20
Screenshot from 2023-10-30 19-41-56

## Additional Information

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #{issue number}_

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

### CODE REVIEW

**Comment by ZakGriffith** [comment]
Requested @escottalexander as there are some conflicts here and he has better context than I.

**Comment by KcPele** [comment]
I have resolved the conflict

**Comment by KcPele** [comment]
> Requested @escottalexander as there are some conflicts here and he has better context than I. Was this from an older version of the branch?
> 
> @KcPele

No, it was the latest update

**Comment by ZakGriffith** [comment]
Gotcha.  Feel free to take the look at eh conflicts if you want.  Otherwise we can wait for the review.

**Comment by escottalexander** [comment]
@ZakGriffith @KcPele Can either of you see why the deployment is failing? Perhaps there are still issues. I can't seem to see the output of the deployment.

File: packages/nextjs/components/lists/SharedProjects.tsx
Comment (escottalexander) [inline packages/nextjs/components/lists/SharedProjects.tsx:97]: The images are still not showing up in this component.

**Review by escottalexander — CHANGES_REQUESTED** [review]
This is very close. Most of the issues are covered but there are a few changes that will need to be made in addition to fixing the deployment pipeline (if it's failure is related to your changes)

**Comment by ZakGriffith** [comment]
It is only failing because of linting errors, which I can clean up after merge

File: packages/nextjs/components/lists/SharedProjects.tsx
Comment (KcPele) [reply packages/nextjs/components/lists/SharedProjects.tsx:97]: this is because there is no image for now, once there is image, it will work

File: packages/nextjs/types/list.ts
Comment (KcPele) [reply packages/nextjs/types/list.ts:25]: Some of the data do have handles, which is why it is optional

File: packages/nextjs/components/project/ProjectHeader.tsx
Comment (escottalexander) [inline packages/nextjs/components/project/ProjectHeader.tsx:34]: This component is currently missing the ability to add a project to the ballot. I am overlooking these commented out lines thinking that they may be related. We should make an issue for restoring this functionality but it is not a problem with your PR specifically.

File: packages/nextjs/components/projects/AllProjects.tsx
Comment (escottalexander) [inline packages/nextjs/components/projects/AllProjects.tsx:20]: If we aren't using these lines lets remove them.

File: packages/nextjs/components/projects/AllProjects.tsx
Comment (escottalexander) [inline packages/nextjs/components/projects/AllProjects.tsx:31]: if we aren't using these lines then lets remove them.

File: packages/nextjs/types/list.ts
Comment (escottalexander) [reply packages/nextjs/types/list.ts:22]: There is no "handle" in the data we are getting for the projects. I believe I removed all references to the handle in the main branch. If I missed somewhere it should be removed along with this interface.

**Review by escottalexander — CHANGES_REQUESTED** [review]
Left a few more comments. let me know if you have any questions.

**Comment by ZakGriffith** [comment]
In testing this looks good to me!  Although we will have to figure out how to move forward getting the icon image to the lists and ballot.

File: packages/nextjs/components/project/ProjectHeader.tsx
Comment (KcPele) [reply packages/nextjs/components/project/ProjectHeader.tsx:34]: I will work on it and make sure the button is working

**Pull request merged**
